src/Tickets/Emails/Email_Handler.php

```diff
@@ -27,6 +27,15 @@ class Email_Handler extends \tad_DI52_ServiceProvider {
 	 */
 	const POSTTYPE = 'tec_tickets_emails';
 
+	/**
+	 * Version number for custom post types.
+	 * 
+	 * @since TBD
+	 * 
+	 * @var string
+	 */
+	const VERSION = '1.0.0';
```

Comment:
I think my comments ended up confusing you here.

The version should come from each email individually as before the most recent changes, and we could use the plugin version number (set it as TBD). That way, we'll be able to update/perform update actions on each email individually rather than in all of them. What I mentioned is that we probably could avoid having that as a class variable.